Source/Utils/WarpXConst.H

```diff
@@ -39,4 +46,6 @@ namespace PhysConst
     static constexpr auto kb   = static_cast<amrex::Real>( 1.380649e-23 );  // Boltzmann's constant, J/K (exact)
 }
 
+extern std::map<std::string, amrex::Real> warpx_constants;
```

Comment:
Discussed today:
- needs a docstring
- idea: allow to overwrite this global from Python

Currently: we allow to overwrite this already from ParamParse.

Idea: avoid global, maybe as a static in the WarpX class or a namespace.